lib/elixir/lib/record.ex

```diff
@@ -36,6 +36,10 @@ defmodule Record do
 
   """
 
+  @type type :: :defrecord | :defrecordp
+  @type name :: atom
+  @type tag :: atom
```

Comment:
Other libraries can just use `tag :: atom` though? I don't see the value of this especially since I never came across any libraries that could have benefitted from it myself. 😕 If someone comes up with the need for it we can revisit maybe?